### ISSUE TITLE
Manifest.js And Vendor.js Not Regenerated When Watch Triggers Build

### ISSUE THREAD

**Issue by slim12kg** [JeffreyWay/laravel-mix#833]
Whenever i run `npm run watch` , any changes in my js files fire the script to rebuild and that works fine, but the manifest.js and vendor.js is always deleted off. And i would have to exit the watch from terminal and run the command again before they are being generated. That behaviour is really wierd to me, at first i thought it only occurs when i have a bug in my code, but it happens if when i don't. It really hurt my feeling    

Here is my webpack.mix.js setup  

```let mix = require('laravel-mix');  

mix.js(['build/main.js','build/index.js'], 'app/js/main.js')
    .extract(['vue','vuex','bulma','jquery'])
    .standaloneSass('build/css/index.scss','app/css')
    .copy('node_modules/ionicons/dist/fonts','app/fonts',false)
    .styles([
        'node_modules/bulma/css/bulma.css',
        'node_modules/ionicons/dist/css/ionicons.css',
        'node_modules/hint.css/hint.css'
    ],'app/css/main.css');  
```

Here is my package.json setup  

```{
  "private": true,
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "dev": "cross-env NODE_ENV=development webpack --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
    "watch": "cross-env NODE_ENV=development webpack --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "bulma": "^0.4.1",
    "cross-env": "^3.2.3",
    "hint.css": "^2.5.0",
    "ionicons": "^3.0.0",
    "jquery": "^3.1.1",
    "laravel-mix": "^0.12.0",
    "lodash": "^4.17.4",
    "vue": "^2.3.3",
    "vueify": "^9.4.1",
    "vuex": "^2.3.1"
  }
}
```

**Comment by teepluss** [comment]
I have the problem.

**Comment by slim12kg** [comment]
@teepluss You found any fix yet. @JeffreyWay  Any help with this  pls?

**Comment by JeffreyWay** [comment]
Fixed in 1.0.
